mitmproxy/websocket.py

```diff
@@ -92,6 +92,16 @@ def set_state(self, state: WebSocketMessageState) -> None:
         ) = state
         self.type = Opcode(typ)
 
+    def format_ws_message(self):
+        message = self.content
+        prefix = b""
+        if self.from_client:
+            prefix = b"[OUTGOING]"
+        else:
+            prefix = b"[INCOMING]"
```

Comment:
Is this some known standard, or is this something we're making up here?

In the latter case I'd favor making this `_format_ws_message` (and same below), I don't want to commit here to a public API prematurely.

Reply:
Is this some known standard, or is this something we're making up here?